### DISCUSSION TITLE
Currency converter 💰

### DISCUSSION THREAD

**Discussion by n8thangreen** [HTAhackathon2024/.github#10] [Ideas]
### What is the problem we're trying to solve

cost are in different currency

### How will we tackle the problem?

-  get data
- other package

### Which tools will we use?

rcost package
[rcost](link)

**Comment by n8thangreen** [comment]
i like this :)

**Comment by RobertASmith** [comment]
More detail needed! :)

**Comment by CiaranEONeill** [comment]
There are sites that allow for costs to be updated adjusting for purchasing power parity and inflation. I've found this useful for systematic reviews and could be used to help plug in values for a CEA. The site I've tended to use is https://eppi.ioe.ac.uk/costconversion/default.aspx 
How this could be done in R is beyond me though.

**Reply by n8thangreen** [reply]
Thanks. This is the associated [paper](https://github.com/user-attachments/files/16453733/cost.webbased.tool.pdf)

This does exactly the sort of thing I tried to do with my `{QALY}` package. It would be good to compare and possibly make a shiny app for it..

**Reply by CiaranEONeill** [reply]
I'll have to go and read the QALY package material - there is a whole world of R that's new to me Nathan

**Comment by tkamiya-hiqa** [comment]
I've toyed with the idea of building a Shiny-based calculator for CPI-PPP adjustment: 
![image](https://github.com/user-attachments/assets/96c17927-9cc2-4508-8db2-abea8c9954f6)

I stopped developing it since I needed to prioritise other tasks, but I thought an app like this would be very useful if data integration could be automated (as opposed to having to constantly update individual CSV files downloaded from different websites).
